<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitConverter</class>
 <widget class="QWidget" name="UnitConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unit Converter</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_Distance">
    <attribute name="title">
     <string>Distance</string>
    </attribute>
    <widget class="QLabel" name="lbl_DistanceFrom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>From</string>
     </property>
    </widget>
    <widget class="QComboBox" name="com_DistanceFrom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="com_DistanceTo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_DistanceTo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>To</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_DistanceFrom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_DistanceTo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Wavelength">
    <attribute name="title">
     <string>Wavelength</string>
    </attribute>
    <widget class="QComboBox" name="com_WavelengthTo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="com_WavelengthFrom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_WavelengthFrom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>From</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_WavelengthTo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>To</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_WavelengthFrom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_WavelengthTo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_Close">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Convert">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
